### PR TITLE
port over remaining grunt tasks

### PR DESCRIPTION
**Pull request by gkatsev**
adds sandbox script and ported over check-translations script.
Contains the code from https://github.com/videojs/video.js/pull/5633/

Looking to see if cdn-links was actually being used and if it's still useful...

### CODE REVIEW

**Comment by gkatsev** [comment]
Failing because of stale package-lock

**Comment by gkatsev** [comment]
Looks like cdn-links was no longer used, sorry for the trouble. removing.

File: package.json [hunk 1]
```diff
@@ -30,39 +32,41 @@
...
     "test:node-require": "node test/require/node.js",
     "test:a11y": "node build/test-a11y.js",
     "test:unit": "karma start test/karma.conf.js",
     "docs": "npm-run-all clean docs:lint docs:api",
     "docs:api": "jsdoc -c .jsdoc.json",
     "postdocs:api": "node ./build/fix-api-docs.js",
-    "netlify": "babel-node ./build/netlify-docs.js",
     "docs:lint": "remark -- './{,!(node_modules)/**/}!(CHANGELOG)*.md'",
     "docs:fix": "remark --output -- './{,!(node_modules)/**/}!(CHANGELOG)*.md'",
+    "docs:lang": "node build/translations.js",
```
Comment (BrandonOCasey) [inline package.json]: we could have this run via lint-staged, whenever translations are changed

Comment (gkatsev) [reply package.json]: Yeah, maybe. Let's do it in another PR, though.

**Pull request merged**